README.md

```diff
@@ -5,7 +5,7 @@ It contains a README file, html, css, gitignore and test.md
 
 # Built With
 **Tech stack**
-client:
+**client:**
   HTML and CSS
 
 # Prerequisities:
```

Comment:
- [OPTIONAL] I would recommend you to make your customisation of the README template in such a way that all the respective titles you need should remain as they are, remove those that are not concern with a project at some point and supply necessary information under each subtitle. With that, your README file would look professional, instructive and easy to use by the users of your project.